content/install-ubuntu.md

```diff
@@ -124,7 +125,7 @@ This is when you will create your first user in your new install of Ubuntu.
 
 ### Slideshow
 
-![Slideshow](/images/install-ubuntu/install-ubuntu-21.04-7.png)
+![Slideshow](/images/install-ubuntu/install-ubuntu-22.04-7.png)
```

Comment:
I see the new image added, but the old one was not removed. Should we get rid of the old one to save space?